collector/bonding_linux.go

```diff
@@ -67,6 +67,9 @@ func readBondingStats(root string) (status map[string][2]int, err error) {
 	status = map[string][2]int{}
 	masters, err := ioutil.ReadFile(path.Join(root, "bonding_masters"))
 	if err != nil {
+		if os.IsNotExist(err) {
```

Comment:
I'd expect this error handling in the `Update` method and not here, it might surprise people / cause panics otherwise as a `nil` map is unexpected when error is `nil`.